README.old.md

```diff
@@ -0,0 +1,2 @@
+# mathMagicians
```

Comment:
- [ ] If this file is not necessary, please delete it from your repository. We don't want unnecessary files as this can also downgrade the professionalism of the repository.

Reply:
This issue is well resolved! ✔️